### PR TITLE
Adding Rahmani

### PR DESCRIPTION
**Pull request by hadirah87**
I added myself to the group webpage.

### CODE REVIEW

File: content/authors/Rahmani/_index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+# Display name
+title: Rahmani
```
Comment (gfudenberg) [inline content/authors/Rahmani/_index.md:3]: maybe change this to Hadi Rahmani?

File: content/authors/Rahmani/_index.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+# For available icons, see: https://sourcethemes.com/academic/docs/page-builder/#icons
+#   For an email link, use "fas" icon pack, "envelope" icon, and a link in the
+#   form "mailto:your-email@example.com" or "#contact" for contact widget.
+social:
+
+#- icon: envelope
+#  icon_pack: fas
+#  link: 'mailto:test@example.org'
+
+- icon: twitter
```
Comment (gfudenberg) [inline content/authors/Rahmani/_index.md:29]: you can delete the twitter icon if you don't want to link to it

File: content/authors/Rahmani/_index.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+highlight_name: false
+
+# Organizational groups that you belong to (for People widget)
+#   Set this to `[]` or comment out if you are not using People widget.
+user_groups:
+- Researchers
+---
+
+Hadi Rahmaninejad is a Postdoctoral Scholar in the Department of [Quantitative and Computational Biology](https://dornsife.usc.edu/qcb/) at USC.  Hadi received a Ph.D. in Physics from Virginia Tech, focusing on biological physics and soft materials. 
+
```
Comment (gfudenberg) [inline content/authors/Rahmani/_index.md:58]: feel free to extend or add links to your previous groups-- up to you

File: content/authors/Rahmani/_index.md (hunk 1 above)
Comment (hadirah87) [reply content/authors/Rahmani/_index.md]: Done.

File: content/authors/Rahmani/_index.md (hunk 2 above)
Comment (hadirah87) [reply content/authors/Rahmani/_index.md]: Done.

File: content/authors/Rahmani/_index.md (hunk 3 above)
Comment (hadirah87) [reply content/authors/Rahmani/_index.md]: Done. Thanks for the suggestion.

**Review by gfudenberg — APPROVED** [review]
thanks, looks good!

**Pull request merged**